dsp-meta-cmd/src/main-server.rs

```diff
@@ -67,7 +67,7 @@ async fn init_server() {
 
     let base_url = settings
         .get::<Url>("base_url")
-        .unwrap_or(Url::parse("http://localhost:3000").unwrap());
+        .unwrap_or(Url::parse("https://meta.dasch.swiss").unwrap());
```

Comment:
will this change affect the local development?

Reply:
we currently have not possibility to provide a configuration to the deployment. It is hardcoded in the Go service as well, e.g. https://meta.stage.dasch.swiss/robots.txt

Reply:
As a followup I have created DEV-3935